### ISSUE TITLE
Zygotic cleavage failure

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6468]
**Preferred term label:**
Zygotic cleavage failure

**Synonyms**
Failure of zygotic cell division

**Definition (free text, please give PubMed ID)**
Failure of a fertilized oocyte to undergo the first round of cell division. 
Comment: A zygote undergoes rapid cell divisions referred to as cleavage because they are not  are not associated with significant growth of the embryo. The iniital round of cleavage forms a spherical ball of cells called the blastula, which then develops into a blastocyst.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0001197
add
-- Abnormal embryonic development
---abnormal preimplantation embryonic development

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619009
4/4 PMID:32502391 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
